template/cmd_main.tmpl

```diff
@@ -23,5 +23,7 @@ func main() {
 		Provider:    fromenv.PubSubProvider(),
 	})
 
-	cmd.Execute()
+	go cmd.Execute()
```

Comment:
This probably isn't what you want, as non server cmd's won't exit until you hit the shutdown hook, I would move the shootdown hook just into the run cmd

Reply:
Correct. I simplified the complete basecommand ending up in a single `lile.Run()` for the service command. ;) I probably looked at the code too much at the time.

Reply:
Example:
```
var upCmd = &cobra.Command{
	Use:   "up",
	Short: "up runs both RPC and pubub subscribers",
	Run: func(cmd *cobra.Command, args []string) {
		lile.Run()
	},
}
```